packages/@webex/plugin-meetings/test/unit/spec/multistream/remoteMediaManager.ts

```diff
@@ -663,6 +663,56 @@ describe('RemoteMediaManager', () => {
       remoteMediaManager.stop();
     });
   });
+
+  describe('setPreferLiveVideo', () => {
+
+    it('sets preferLiveVideo to true', async () => {
+      const config = cloneDeep(DefaultTestConfiguration);
+
+      config.video.initialLayoutId = 'OnePlusFive';
+
+      remoteMediaManager = new RemoteMediaManager(
+        fakeReceiveSlotManager,
+        fakeMediaRequestManagers,
+        config
+      );
+  
+      await remoteMediaManager.start();
+  
+      resetHistory();
+      remoteMediaManager.setPreferLiveVideo(true);
+
+      expect(config.video.preferLiveVideo).to.equal(true);
+
+      Object.values(remoteMediaManager.media.video.activeSpeakerGroups).forEach((activeSpeakerGroup: any) => {
+        expect(activeSpeakerGroup.options.preferLiveVideo).to.equal(true);
+      });
+    });
+
+    it('sets preferLiveVideo to false', async () => {
```

Comment:
same as with RemoteMediaGroup tests -  I don't see much value in doing same test with true and false, but if you do then please make it a parametrized one to avoid duplication. More useful is to test if new value is same or different than the old one